README.md

```diff
@@ -1,55 +1,53 @@
-level-mem
-=========
+# level-mem
 
-<img alt="LevelDB Logo" height="100" src="http://leveldb.org/img/logo.svg">
+> A convenience package bundling `levelup` and `memdown`
 
-**A convenience package bundling LevelUP & MemDOWN**
+[![level badge][level-badge]](https://github.com/level/awesome)
+[![npm](https://img.shields.io/npm/v/level-mem.svg)](https://www.npmjs.com/package/level-mem)
+[![Build Status](https://secure.travis-ci.org/Level/mem.png)](http://travis-ci.org/Level/mem)
+[![david](https://david-dm.org/Level/mem.svg)](https://david-dm.org/level/mem)
+[![JavaScript Style Guide](https://img.shields.io/badge/code_style-standard-brightgreen.svg)](https://standardjs.com)
+[![npm](https://img.shields.io/npm/dm/level-mem.svg)](https://www.npmjs.com/package/level-mem)
 
-[![NPM](https://nodei.co/npm/level-mem.png)](https://nodei.co/npm/level-mem/)
+This is a convenience package that bundles the current release of [`levelup`](https://github.com/level/levelup) and [`memdown`](https://github.com/level/memdown) and exposes `levelup` on its export.
 
-[![Build Status](https://secure.travis-ci.org/Level/level-mem.png)](http://travis-ci.org/Level/level-mem) [![Greenkeeper badge](https://badges.greenkeeper.io/Level/level-mem.svg)](https://greenkeeper.io/)
-
-This is a convenience package that bundles the current release of **[LevelUP](https://github.com/rvagg/node-levelup)** and **[MemDOWN](https://github.com/rvagg/node-memdown)** and exposes LevelUP on its export.
-
-Use this package to avoid having to explicitly install MemDOWN when you want to use MemDOWN with LevelUP for non-persistent LevelUP data storage.
+Use this package to avoid having to explicitly install `memdown` when you want to use `memdown` with `levelup` for non-persistent `levelup` data storage.
 
 ```js
-var level = require('level-mem')
+const level = require('level-mem')
 
 // 1) Create our database, supply location and options.
 //    This will create or open the underlying LevelDB store.
-var db = level('w00t')
+const db = level()
```

Comment:
@vweevers I'm assuming you meant this `location`

Reply:
Exactly. And the comment above it ("supply location")

Reply:
Oh right.